packages/grafana-ui/src/components/TimeSeries/utils.ts

```diff
@@ -304,8 +314,16 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<{
 
       let incrs: uPlot.Axis.Incrs | undefined;
 
+      // TODO: these will be dynamic with frame updates, so need to accept getYTickLabels()
```

Comment:
this comment is no longer true if we decide that the axis should always contain all possible values of all enum fields, rather than having ticks for just the values that exist in the enum field data.